sync/src/test-support/java/tech/pegasys/artemis/sync/SyncingNodeManager.java

```diff
@@ -60,6 +66,7 @@ public static SyncingNodeManager create(
       Consumer<Eth2P2PNetworkBuilder> configureNetwork)
       throws Exception {
     final EventBus eventBus = new EventBus();
+    final EventChannels eventChannels = new EventChannels(new NoOpMetricsSystem());
```

Comment:
Would probably be better to use the existing constructor here and specify the exception handler explicitly.  We also don't want to wind up using an async event channel - the `EventBus` in this case is single threaded so we want to use sync event channels.

Reply:
Done.